client/src/components/SubscriberTable/SubscriberTable.jsx

```diff
@@ -45,11 +45,11 @@ const SubscriberTable = (props) => {
                     <div className="text-sm text-gray-900">{subscriber.email}</div>
                     </td>
                     <td className="px-6 py-4 whitespace-nowrap">
-                      <StatusPill value={subscriber.status} />
+                      <StatusPill value={subscriber.subscribed} />
```

Comment:
column on subscribers table and attribute on Subscriber model is called subscribed not status